thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -49,6 +67,64 @@ export default Component.extend({
     }
   ),
 
+  /**
+   * Information about an anomaly's baselines, changes, and values to be displayed in the anomaly overview
+   * @type {Object}
+   * @example
+   * {
+   *   current: {
+   *    change: 1.1,
+   *    changeFormatted: +1.1,
+   *    value: 5000
+   *   }, {
+   *   baseline: {
+   *    change: 1.1,
+   *    changeFormatted: 1.1,
+   *    value: 1000
+   *   }
+   * }
+   */
+  anomalyInfo: computed(
+    'aggregates',
+    'anomalyUrns',
+    function () {
+      const { aggregates, anomalyUrns } = getProperties(this, 'aggregates', 'anomalyUrns');
+
+      const offsets = ['current', 'baseline', 'wo1w', 'wo2w', 'wo3w', 'wo4w'];
+      const metricUrns = filterPrefix(anomalyUrns, 'thirdeye:metric:');
+
+      if (!metricUrns) { return; }
+
+      let anomalyInfo = {};
+
+      [...offsets].forEach(offset => {
+        const change = aggregates[toOffsetUrn(metricUrns[0], 'current')] / aggregates[toOffsetUrn(metricUrns[0], offset)] - 1;
+        const roundedChange = (Math.round(change * 1000) / 10.0).toFixed(1);
```

Comment:
you could use the "humanizeChange()" util (if it's already merged)

Reply:
Will do. I'll still need to calculate the raw change to compute the color of the number in addition to computing the human-readable value that gets returned from `humanizeChange`, just to be clear.